app/collections/get-favorites-name.cjsx

```diff
@@ -0,0 +1,5 @@
+# Get the name of the favorites collection for a project
```

Comment:
If favorites collections are per-project, couldn't the name always just be "Favorites"? Project names can change.